site/tsdb-engine/css/demo.css

```
@@ -161,3 +161,562 @@
   color: var(--ink);
   text-align: right;
 }
+
+/* ── Dataset Panel ──────────────────────────────────────────────────── */
+
+.dataset-panel { }
```

Stylelint (17.7.0)

[error] Empty block (block-no-empty)

(block-no-empty)